🔨 Add Portuguese translations LLM prompt

ceb10n opened the pull request:
* add llm-prompt.md for portuguese translations
* add missing translations to validate effectiveness (`_llm_test.md` and `migrate-from-pydantic-v1-to-pydantic-v2.md`)

ceb10n committed:
add: portuguese llm-prompt

ceb10n committed:
add _llm-test and missing translations to validate effectiveness

pre-commit-ci[bot] committed:
🎨 [pre-commit.ci] Auto format from pre-commit.com hooks

ceb10n committed:
Enhance Portuguese translation guidelines

Added guidance for imperative translations and a new entry for 'Very Technical Details'.

victorgaard reviewed (approved):
That looks great, thanks for setting it up! 🚀

ceb10n committed:
Add non-Annotated to llm-prompt

added non-Annotated instruction

ceb10n committed:
Enhance translation guidelines for Portuguese documentation

Added guidelines for translating terms and phrases into Portuguese, including specific instructions on maintaining existing translations and using neutral language.

ceb10n committed:
Add translation for 'you should' in LLM prompt

ceb10n committed:
Update translation for 'operation IDs'

ceb10n committed:
remove migrate-from-pydantic-v1-to-pydantic-v2.md

The new file was added on the PR https://github.com/fastapi/fastapi/pull/14228

tiangolo reviewed (approved):
Awesome, thank you! 🚀
